module3/src/MyList.java

```diff
@@ -0,0 +1,697 @@
+import java.util.*;
+
+/**
+ * Created by yurii.pyvovarenko on 11/28/2021.
+ */
+public class MyList<T> implements List<T>, Iterable<T> {
+    private static final Object[] DEFAULTCAPACITY_EMPTY_ELEMENTDATA = {};
```

Comment:
Naming exception. Should be DEFAULD_CAPACITY_EMPTY_ELEMENT_DATA. Each word should be separate with '_' separator

Reply:
corrected to DEFAULD_CAPACITY_EMPTY_ELEMENT_DATA